src/gui/virtualstudio.cpp

```diff
@@ -866,9 +898,17 @@ void VirtualStudio::getServerList(bool firstLoad, int index)
                 VsServerInfo* serverInfo = new VsServerInfo(this);
                 serverInfo->setIsManageable(
                     servers.at(i)[QStringLiteral("admin")].toBool());
-                QString status = servers.at(i)[QStringLiteral("status")].toString();
-                // Only include active servers, or servers that we can manage.
-                if (status == QLatin1String("Ready") || serverInfo->isManageable()) {
+                QString status     = servers.at(i)[QStringLiteral("status")].toString();
+                bool activeStudio  = status == QLatin1String("Ready");
+                bool managedStudio = servers.at(i)[QStringLiteral("managed")].toBool();
```

Comment:
Why create a managedStudio variable here? Why not just use serverInfo->isManageable()?

Reply:
The meaning is different between these two, it looks like `serverInfo->isManageable()` is based on the `admin` key of the server object whereas this `managedStudio` variable is based on the `managed` key.

The `admin` field is a boolean indicating whether your current user has been given permission to control all the knobs that VS supports. The `managed` field is true if the server is managed by our cloud infra, and false if it's a self-hosted server (see step 2 on https://app.jacktrip.org/studios/create). It's admittedly confusing but `managed: false` would mean the server is self-hosted which is what the toggle intends to control.

Reply:
I think I'll change the variable name to `hostedStudio` to avoid any confusion

Reply:
I misread that. I thought it was using the same key but just saw that the manageable is taken from the admin setting. Yes changing to hostedStudio would make more sense.